posthog/management/commands/sync_persons_to_clickhouse.py

```diff
@@ -0,0 +1,151 @@
+from uuid import UUID
+
+import structlog
+from django.core.management.base import BaseCommand
+
+from posthog.client import sync_execute
+from posthog.models.person import PersonDistinctId
+from posthog.models.person.person import Person
+from posthog.models.person.util import _delete_ch_distinct_id, create_person, create_person_distinct_id
+
+logger = structlog.get_logger(__name__)
+
+PERSON_TABLE = "person"
+DISTINCT_ID_TABLE = "persondistinctid"
+
+
+class Command(BaseCommand):
+    help = """Sync person or distinct id tables from postgres to ClickHouse.
+        Lookup from Postgres and with a lower version in ClickHouse will be updated.
+        Note higher versions in ClickHouse will be ignored.
+        Recommended: run first without `--live-run` and first for person table, then distinct_id table
+        """
+
+    def add_arguments(self, parser):
+        parser.add_argument("--team-id", default=None, type=int, help="Specify a team to fix data for.")
+        parser.add_argument(
+            "--tables", default=[PERSON_TABLE, DISTINCT_ID_TABLE], action="append", help="Which tables to sync"
```

Comment:
nit: I see how this is more general and allows more tables to be added but I think it's unnecessary complexity like validating the set sizes and all. Easiest would be a `store_true` for each of `--person-distinct-id` and `--person`.